CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and **Merged pull requests**. Critical items to know are:
 The versions coincide with releases on pip. Only major versions will be released as tags on Github.
 
 ## [0.0.x](https://github.com/singularityhub/sregistry-cli/tree/master) (0.0.x)
+ - sregistry.main.s3.pull: add support for setting object ACL (0.2.26)
+   - sregistry.main.s3.query: catch exceptions when objects are not readable
  - restore quiet to False after initial setting to True in inspect (0.2.25)
```

Comment:
I can handle this merge conflict (master is at 0.2.26 so this should be merged and then changed to 0.2.27) but for version.py, the bump should be up to 0.0.27! see -> https://github.com/singularityhub/sregistry-cli/blob/master/sregistry/version.py#L11

Reply:
Apologies, forgot to rebase against origin/master!

Will fix right away!